### DISCUSSION TITLE
502 Bad Gateway after Update

### DISCUSSION THREAD

**Discussion by Tzzui** [mediacms-io/mediacms#849] [Q&A]
**Describe the issue**
Hello,
I'm getting 502 Bad Gateway Errors after updating mediacms

**To Reproduce**
Follow the update instructions:

cd /path/to/mediacms/installation
docker pull mediacms/mediacms
docker-compose down
docker-compose up

**Expected behavior**
Mediacms should not give a 502 error

**Screenshots**
-

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - Installation method: Docker install (with letsencrypt)
 - Browser: Firefox

**Additional context**

**Comment by mgogoulos** [comment]
Have a look on the update section, on https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#3-docker-installation
There's backward incompatibility between Posgresql versions, but two workarounds are presented there

**Reply by Tzzui** [reply]
edit: I have now updated the container to postgres 15, imported the sql dump from the previous version and pulled the latest mediacms image. The issue persists. No errors upon starting the containers, but I still get the 502 Bad Gateway error. 

psql version is 15 and the database seems to be intact:
![image](https://github.com/mediacms-io/mediacms/assets/71082590/91fd7827-79a2-48a4-84cb-354f481e0d38)


----
Thank you for the quick reply, I don't know what to edit in the compose file to make use of the existing image, though. Could you elaborate (if it's not too much trouble)? 

I also did not get the error described in that section. The db_1 container went up just fine.

**Reply by mgogoulos** [reply]
Hi, I would expect that changing the version on docker-compose.yml is easier, instead of the value

```
    image: postgres:15.2-alpine
```

change to 

```
    image: postgres:13
```

**Reply by mgogoulos** [reply]
also cannot provide any other input unless i can see log messages. Check out with 

```
docker-compose logs db
docker-compose logs web
```

and see share the output
